ff/src/biginteger/mod.rs

```diff
@@ -513,21 +514,15 @@ impl<const N: usize> FromBytes for BigInt<N> {
     }
 }
 
-impl<const N: usize> UpperHex for BigInt<N> {
```

Comment:
Wasn't that meant to stay?

Reply:
sure! I will seperate it in two PR with BigInt<N>... my bad (this time :-D)